README.md

```diff
@@ -112,8 +122,8 @@ end
 
 Your answer:
 ```text
-
-Replace this with your answer
+describe wraps a bunch of test into one functionality
+contexts wraps test around the same state
```

Comment:
describe and context are the same under the hood - http://lmws.net/describe-vs-context-in-rspec